backend/backend/src/main/java/de/aaaaaaah/velcom/backend/storage/db/DatabaseStorage.java

```diff
@@ -76,40 +79,55 @@ private void migrate(String jdbcUrl) {
 	}
 
 	/**
-	 * @return a {@link DSLContext} instance providing jooq functionality along with a connection to
-	 * 	the database
+	 * Acquires read access to the database.
+	 * @return a way to interact with the database in a read only way
```

Comment:
A lot of the javadoc in this file seems to not have been auto-formatted.

Reply:
Hopefully fixed in https://github.com/IPDSnelting/velcom/commit/6f1f5854816740d8c31bc66b55f552522aa5bafc